File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacPersistance/DAO/MonedasDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface MonedasDAO
    {
        int Insertar(MonedasDTO moneda);
        int Modificar(MonedasDTO moneda);
        int eliminar(MonedasDTO moneda);
        MonedasDTO ObtenerPorId(int monedaId);
        IList<MonedasDTO> ListarTodos();
    }
}

File: Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/MonedasDAO.java
package pe.edu.pucp.softpac.dao;

import java.util.List;
import pe.edu.pucp.softpac.model.MonedasDTO;

public interface MonedasDAO {
    public Integer insertar(MonedasDTO moneda);
    
    public MonedasDTO obtenerPorId(Integer monedaId);
    
    public List<MonedasDTO> listarTodos();
    
    public Integer modificar(MonedasDTO moneda);
    
    public Integer eliminar(MonedasDTO moneda);

    public MonedasDTO obtenerPorDivisa(String divisaMoneda);

}
